### PR TITLE
Prize component

### PR DESCRIPTION
**Pull request by kristelfung**
4 column for prizes

### CODE REVIEW

File: src/js/modules/core/components/Prizes.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import * as React from "react";
+import injectSheet, {WithStyles} from "react-jss";
+import { Theme } from "../../ThemeInjector";
+
+const styles = (theme: Theme) => ({
+  prizes: {
+    backgroundColor: theme.backgroundColor,
+    width: "100%",
+    display: "flex",
```
Comment (NicholasLYang) [inline src/js/modules/core/components/Prizes.tsx]: Why not grid?

Comment (kristelfung) [reply src/js/modules/core/components/Prizes.tsx]: columns came out strangely narrow when I tried grid. I will try grid again then

Comment (kristelfung) [reply src/js/modules/core/components/Prizes.tsx]: just kidding it works, reverting to grid!

Comment (themichaelyang) [reply src/js/modules/core/components/Prizes.tsx]: if we're using grid, let's make sure we have a polyfill or something:

https://caniuse.com/#feat=css-grid (87% isn't great)

**Comment by themichaelyang** [comment]
I'm still not in favor of using grid (see above for browser support). I did some research and there doesn't seem to be a really stable polyfill (additionally, this layout can be quickly done with flexbox or even percentage block elements). 

If you really want to use grid, try adding this to the site:
https://github.com/FremyCompany/css-grid-polyfill

**Comment by themichaelyang** [comment]
Closing, since this is a forked branch (I can't commit). See the new PR.

**Pull request closed without merging**